packages/jobs/lib/runner/runner.ts

```diff
@@ -29,7 +29,7 @@ export function getRunnerId(suffix: string): string {
 
 export async function getOrStartRunner(runnerId: string): Promise<Runner> {
     const waitForRunner = async function (runner: Runner): Promise<void> {
-        const timeoutMs = 5000;
+        const timeoutMs = isEnterprise ? 60000 : 5000;
```

Comment:
not sure this is necessary since for remote runner we can assume it is always on (ie: it is not managed by nango)